src/uwtools/config/formats/ini.py

```diff
@@ -81,13 +81,9 @@ def dump_dict(path: OptionalPath, cfg: dict, opts: Optional[ns] = None) -> None:
         parser = configparser.ConfigParser()
         s = StringIO()
         cfgdepth = depth(cfg)
-        assert cfgdepth in (1, 2)  # 1 => .sh, 2 => .ini
-        if cfgdepth == 2:
-            parser.read_dict(cfg)
-            parser.write(s, space_around_delimiters=opts.space if opts else True)
-        else:
-            for key, value in cfg.items():
-                print(f"{key}={value}", file=s)
+        assert cfgdepth in (1, 2)  # 2 => .ini
+        parser.read_dict(cfg)
+        parser.write(s, space_around_delimiters=opts.space if opts else True)
```

Comment:
What happens here if `opts` doesn't include a `space` attribute?

Do we want to stick with the optional namespace object for these optional args, or should we go with the Python-standard kwargs? This would (IMO) simplify to:

```
parser.write(s, space_around_delimiters=kwargs.get('space', True))
```

Reply:
I definitely prefer kwargs, I'll get that in